README.rst

```diff
@@ -0,0 +1,49 @@
+===================
+RDO Project Easyfix
```

Comment:
I suggest naming README.md which github nicely renders as Markdown including links and... use Markdown ;)